src/pages/index.js

```diff
@@ -64,28 +65,44 @@ function createCard(items) {
     function (card) {
       confirmPopup.openPopup();
       confirmPopup.handleConfirm(function () {
+        confirmPopup.handleButtonLoading(true);
         const cardId = items._id;
-        newApi.deleteApiCard(cardId).then(card);
-        card.deleteCard();
+        newApi
+          .deleteApiCard(cardId)
+          .then(() => {
+            card.deleteCard();
+            confirmPopup.closePopup();
+            confirmPopup.handleButtonLoading(false);
```

Comment:
* You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case.

**Please, correct it everywhere**